### ISSUE TITLE
Xiaomi series evaluated as rooted

### ISSUE THREAD

**Issue by HanHyeoksu** [scottyab/rootbeer#63]
I ran the 'RootBeer Sample' app downloaded by google play on some xiaomi devices. and I found a problem that the devices are evaluated as rooted.

```
Xiaomi Mi max2 - busybox binary
Hongmi(Redmi) Note3 Pro - busybox binary, dangerous props
Xiaomi Mi Note - busybox binary, su binary, root native

Xiaomi Mi A1 - not rooted
Huawei honor 6x - not rooted
```

I'm not sure whether the devices are actually rooted or not. But as far as I know the devices are not rooted because I got the devices from where some kind of public institutions.

Is that issues for xiaomi devices? Or is that rooted actually?
How can I deal with it?

**Comment by TheComputerGuy96** [comment]
@HanHyeoksu What ROMs do those Xiaomi devices run?

**Comment by HanHyeoksu** [comment]
@TheComputerGuy96 I don’t know exactly it’s ROM name or not but I remember that ROM name was MIUI. Is there some relationship between ROM and rooting?

**Comment by stealthcopter** [comment]
@HanHyeoksu The busybox binary can sometimes be a false positive when it comes to detecting root, as some manufacturers leave it in their production builds. This is why we have an option to ignore this when checking for root when using this library.

With your Xiaomi Mi Note, the su binary was found which is usually pretty clear indication that rooting is present so might be worth checking that out if you weren't expecting that.

**Comment by HanHyeoksu** [comment]
@stealthcopter Thanks. You're right. I replaced isRooted with isRootedWithoutBusyBoxCheck in my app.
So, you mean that Xiaomi Mi Note is likely to be rooted because su binary was found. right?
Then what about Redmi Note3 pro? 'Dangerous props' was found in it. It means that rootbeer found the 'ro.debuggable=1' and 'ro.secure=0' in the android phone and I've got know the fact that the props is found then it's likely to be rooted.
